core/marker_manager.js

```diff
@@ -24,7 +25,7 @@ goog.requireType('Blockly.WorkspaceSvg');
  * @constructor
  * @package
```

Comment:
The current style (see #5026) would have `@package` moved to a separate JSDoc just above the `exports.MarkerManager = …` line.

Reply:
Done